bgpd/bgp_mplsvpn.c

```diff
@@ -602,8 +602,8 @@ int vpn_leak_label_callback(
 	return 0;
 }
 
-static void sid_register(struct bgp *bgp, const struct in6_addr *sid,
-			 const char *locator_name)
+void sid_register(struct bgp *bgp, const struct in6_addr *sid,
+		  const char *locator_name)
```

Comment:
Is the space is ok here? Format was applied?

Reply:
Yes. This space was added by `clang-format`.

Reply:
Thanks